tuplex/codegen/include/ASTAnnotation.h

```diff
@@ -352,13 +351,20 @@ class Symbol : public std::enable_shared_from_this<Symbol> {
     }
 };
 
+///! iterator-specific annotation: iterator name i.e. from which function the iterator was generated, arguments' type of the function, iteratorInfo of arguments
+struct IteratorInfo {
+    std::string iteratorName;
+    python::Type argsType;
+    std::vector<IteratorInfo *> argsIteratorInfo;
```

Comment:
can we maybe describe this member a bit more? what is it?

Reply:
More comments added.